auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -0,0 +1,296 @@
+            </span>
+          </v-col>
+
+          <template v-if="!isEdit">
+            <v-col
+              cols="6"
+              sm="3"
+              class="text-right"
+            >
+              <v-btn
+                color="primary"
+                class="action-btn px-6"
+                @click="actionBtnClicked()"
+              >
+                {{ actionBtnText }}
+              </v-btn>
+              <span>
+                <v-menu
+                  v-model="menu"
+                  offset-y
+                  nudge-left="90"
+                >
+                  <template #activator="{ on }">
+                    <v-btn
+                      color="primary"
+                      class="more-actions-btn menu-btn px-3"
+                      v-on="on"
+                    >
+                      <v-icon v-if="menu">
+                        mdi-menu-up
+                      </v-icon>
+                      <v-icon v-else>
+                        mdi-menu-down
+                      </v-icon>
+                    </v-btn>
+                  </template>
+
+                  <v-list dense>
+                    <v-list-item @click="triggerEditOnOff()">
+                      <v-list-item-subtitle>
+                        <v-icon>mdi-pencil</v-icon>
+                        <span class="pl-2 edit-txt">Edit</span>
+                      </v-list-item-subtitle>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </span>
+            </v-col>
+          </template>
+        </v-row>
+
+        <!-- Cancel/Save buttons -->
+        <template v-if="isEdit">
+          <v-divider class="mb-8 mt-8" />
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              class="text-right"
+            >
+              <v-btn
+                color="primary"
+                large
+                outlined
+                class="mr-3"
+                @click="triggerEditOnOff()"
+              >
+                Cancel
+              </v-btn>
+              <v-btn
+                color="primary"
+                large
+                @click="saveBtnClicked()"
+              >
+                Save
+              </v-btn>
+            </v-col>
+          </v-row>
+        </template>
+      </article>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue } from 'vue-property-decorator'
+import { mapActions } from 'pinia'
+import { useStaffStore } from '@/stores/staff'
+
+@Component({
+  methods: {
+    ...mapActions(useStaffStore,
+      ['getDissolutionBatchSize',
+        'isDissolutionBatchOnHold',
+        'updateDissolutionBatchSize',
+        'updateDissolutionBatchOnHold'])
+  }
+})
+export default class DissolutionSchedule extends Vue {
+  $refs: {
+    numberOfBusinessesRef: any
+  }
+
+  // Local properties
+  menu = false
+  numberOfBusinesses = 0
+  isEdit = false
+  isOnHold = false
```

Comment:
You may notice that I'm using the onHold a lot. This is because, from the configurations table (got this from Argus), we'll be getting something like this:
```json
{
  "configurations": [
    {
      "name": "NUM_DISSOLUTIONS_ALLOWED",
      "value": "100"
    },
    {
      "name": "DISSOLUTIONS_ON_HOLD",
      "value": false
    },
   ...
  ]
}
```

So, I'm using the on hold term.